src/settings/MappingProfiles/detailsSections/utils.js

```diff
@@ -241,10 +241,10 @@ export const getAccountNumbersByCode = organization => {
   }), {});
 };
 
-export const PERCENTAGE_VALUE = 'Percentage';
+export const PERCENTAGE_VALUE = 'percentage';
 
 export const renderAmountValue = (amountValue, amountType, currency) => {
-  const amountSymbol = amountType === `"${PERCENTAGE_VALUE}"` ?
+  const amountSymbol = amountType?.toLowerCase() === `"${PERCENTAGE_VALUE}"` ?
```

Comment:
`PERCENTAGE_VALUE` is already a string so string interpolation is unnecessary:
```
const amountSymbol = amountType?.toLowerCase() === PERCENTAGE_VALUE ?
```

Also, you don't need to change this now since you have working code, but it does a lot of reinventing the wheel: everything you're doing here and in `<CurrencySymbol>` is already handled by [react-intl's FormattedNumber](https://formatjs.io/docs/react-intl/components#formattednumber). Just use it.

Reply:
@zburke that's data import mapping syntax specific, everything that is not a MARC field like '918$a' should be wrapped in the quotes

Reply:
on UI it looks like it for example, otherwise, you will get an error, because it will check for a mapping syntax
<img width="380" alt="Screenshot 2024-10-11 at 19 55 51" src="https://github.com/user-attachments/assets/4821666e-8933-4f18-a6b3-9671e0aa7071">

Reply:
🤯